.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+  - 2.7
+  - 3.4
+services:
+  - postgresql
+env:
+  -DJANGO=1.10 DB=postgresql
+install:
+  - pip install -r requirements.txt
+before_script:
+  - sudo su - postgres
+  - psql
+  - create database odyssy;
```

Comment:
Commands are run by `bash` in `before_script`. So psql commands won't work. I recommend having a look at https://github.com/anyaudio/anyaudio-server/blob/master/.travis.yml